mdpow/tests/test_emin.py

```diff
@@ -1,10 +1,13 @@
 import mdpow.equil
+import gromacs
 import tempdir as td
 import os
 
+
 class TestEnergyMinimization(object):
     
     def setup(self):
+        gromacs.config.set_gmxrc_environment('~/.conda/envs/MDPOW3/bin/GMXRC')
```

Comment:
should not be needed

Reply:
I'll try without it, I was having some issues with it not finding the conda GMXRC, mostly a temporary solution to get tests running so I could evaluate my code. I defiantly need to remove some of the modifications I made while trying to troubleshoot import errors.